python/python/bystro/prs/handler.py

```diff
@@ -1,10 +1,50 @@
-from bystro.prs.messages import PRSJobData, PRSJobResult
+from pathlib import Path
+from typing import Any, Callable
+
+from msgspec import json
+
 from bystro.beanstalkd.worker import ProgressPublisher
+from bystro.ancestry.ancestry_types import AncestryResults
+from bystro.prs.messages import PRSJobData, PRSJobResult
+from bystro.prs.preprocess_for_prs import generate_c_and_t_prs_scores
+
+
+def make_calculate_prs_scores(
+    cluster_opensearch_config: dict[str, Any]
+) -> Callable[[ProgressPublisher, PRSJobData], PRSJobResult]:
+    def calculate_prs_scores(_publisher: ProgressPublisher, prs_job_data: PRSJobData) -> PRSJobResult:
+        """
+        Calculate PRS scores for a single submission
+        """
+
+        assembly = prs_job_data.assembly
+        dosage_matrix_path = prs_job_data.dosage_matrix_path
+        p_value_threshold = prs_job_data.p_value_threshold
+        disease = prs_job_data.disease
```

Comment:
Instead of disease, I think trait or phenotype might be appropriate i.e. height, BMI, WHR

Reply:
There are 18 places where it says disease, so here and throughout would prefer trait or phenotype